topdown/http.go

```diff
@@ -218,11 +231,20 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 			if !ok {
 				return nil, fmt.Errorf("invalid type for headers key")
 			}
+		case "timeout":
+			timeout, err = parseTimeout(obj.Get(val).Value)
+			if err != nil {
+				return nil, err
+			}
 		default:
 			return nil, fmt.Errorf("invalid parameter %q", key)
 		}
 	}
 
+	client := &http.Client{
+		Timeout: timeout,
+	}
```

Comment:
👌 this should cover the end to end timeout from dial to response body received. nice!